mysql/sql/record.sql

```diff
@@ -0,0 +1,15 @@
+use db;
+
+DROP TABLE IF EXISTS record;
+
+CREATE TABLE record
+(
+    id                INT NOT NULL AUTO_INCREMENT,
+    player_id         INT NOT NULL,
+    stage_id          INT NOT NULL,
+    is_clear          BIT(1),
+    play_times        INT,
```

Comment:
```suggestion
    play_times        INT COMMENT '該当ステージプレイした回数',
```
こんな感じにコメントを追加してほしいです。
IDとか当たり前のやつはいらないと思うので、ちょっと適当に判断してほしい

Reply:
変えました！